src/index.ts

```diff
@@ -466,6 +466,21 @@ function liskAddressDecoder(data: string): Buffer {
 
   return Buffer.from(bigInt(data.slice(0, -1)).toString(16), 'hex');
 }
+  
+function seroAddressEncoder(data: Buffer): string {
+  const address =  bs58EncodeNoCheck(data);
+
+  return address;
+}
+
+function seroAddressDecoder(data: string): Buffer {
+  const bytes = bs58DecodeNoCheck(data);
+  if (bytes.length === 96) {
```

Comment:
Why did you get rid of 64? As yo referenced, it does check the length of bytes of 96 and 64 post decoding. https://github.com/sero-cash/go-sero/blob/master/common/address/account_types.go#L95

Reply:
I was like thinking that it may be what made you to think that my code was wrong.

Reply:
I asked Sero team for the actual specification page and got this one https://wiki.sero.cash/en/index.html?file=Tutorial/sero-exchange-interface#Account

According to the specification, they have `seed(seed) -> sk(Private Key) -> TK(Tracking Key) -> PK(Public Key) -> PKr(Receipt Code)` where the length of the seed is 32,  private key and tk are 64, and public key is 64, and the Pkr is 96.

Reply:
@makoto  Thanks for the update. So should I return to 64 and 96 or just 64?

Reply:
Just confirmed the team that `Pkr` is what we need to store so your current 96 should be fine. I will merge this now. Thank you for your patience.